src/components/client/core/Menu/Menu.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import { Button, Menu } from "components/client";
+import { Text } from "components/universal";
+import { HStack } from "generated/panda/jsx";
+
+import type { Meta, StoryObj } from "@storybook/react";
+import type { MenuItem, MenuItemGroup } from "components/client";
+
+type Story = StoryObj<typeof meta>;
+
+const SUBMENU_GROUP_ITEMS: MenuItem[] = [
+  { id: "item-7", child: "Item 7" },
+  { id: "item-8", child: "Item 8" },
+  { id: "item-9", child: "Item 9" },
+];
+
+const SUBMENU_GROUP: MenuItemGroup = {
+  id: "group-3",
+  label: "Group 3",
+  items: SUBMENU_GROUP_ITEMS,
+};
+
+const GROUP_ONE_ITEMS: MenuItem[] = [
+  {
+    id: "item-1",
+    child: (
+      <HStack justify="space-between" flex={1}>
+        Item 1
+        <Text color="fg.subtle" textStyle="xs">
+          ⇧⌘P
+        </Text>
+      </HStack>
+    ),
+  },
+  { id: "item-2", child: "Item 2" },
+  { id: "item-3", child: "Item 3" },
+];
+
+const GROUP_TWO_ITEMS: MenuItem[] = [
+  { id: "item-4", child: "Item 4" },
+  {
+    id: "item-5",
+    subMenu: true,
```

Comment:
For submenus, do you think we should render an arrow (e.g. a right chevron), or leave it completely up to the user? I think it would be nice to have a default chevron with the ability to override the icon (and nullify it), but would love to hear what you think!

Reply:
If you checkout the `Menu` component itself... without getting too recursive, it gets tough to apply that specifically to that element as it is actually just rendering the `child` which in this case is just another `Menu` component but using `MenuTriggerItem` rather than `MenuTrigger` to pick up the correct recipe styles.. If people want the chevron, they can bake it into their custom implementation of the `MenuTriggerItem` though

Reply:
That's fair, in that case I think we should add a task for it. I just explored a handful of component libraries and it seems to be a trend. Radix for example: https://www.radix-ui.com/primitives/docs/components/menubar

Adding card to Trello right now! No need to worry about it in this PR